src/main/kotlin/javabot/model/Factoid.kt

```diff
@@ -61,6 +61,11 @@ class Factoid(var name: String = "", var value: String = "", var userName: Strin
         if (name.endsWith(" $^")) {
             replaced = urlencode(camelcase(replacedValue))
         }
+
+        if (replaced.isBlank()) {
+            return "You were supposed to provide a target for \"$name\", $sender"
```

Comment:
All responses to the user should go in `sofia.properties` and then this should use the resulting method here and in the tests.